assets/js/translator.js

```diff
@@ -188,6 +217,90 @@ if(modeEnabled('translation')) {
+                    'title': 'Suggestion cannot be empty.',
+                    'trigger': 'manual',
+                    'placement': 'bottom'
+                });
+                $('#suggestedWordInput').tooltip('show')
+                setTimeout(function() {
+                    $('#suggestedWordInput').tooltip('hide');
+                    $('#suggestedWordInput').tooltip('destroy');
+                }, 3000);
+
+                return;
+            }
+
+            // Obtaining context, (± config.SUGGESTIONS.context_wrap) words
+            // fallback to complete text if this fails.
+            var hashedWord = fromWord.hashCode() + fromWord + fromWord.hashCode();
+            $('#wordGettingSuggested').text(hashedWord);
+
+            var splitted = $('#translatedText').text().split(' ');
+            $('#wordGettingSuggested').text(fromWord)
+
+            var targetIndex = splitted.indexOf(hashedWord);
+            var wrapLength = parseInt(config.SUGGESTIONS.context_wrap);
+            var begin, end;
+            begin  = (targetIndex > wrapLength) ? (targetIndex - wrapLength) : 0;
+            end = (splitted.length - targetIndex - 1 > wrapLength) ? (targetIndex + wrapLength + 1) : splitted.length;
+            
+            var context = splitted.slice(begin, end).join(' ').replace(hashedWord, fromWord);
+            if (!context) {
+                context = $('#translatedText').attr('pristineText');
+            }
+
+            $.ajax({
+                url: config.APY_URL + '/suggest',
+                type: 'POST',
+                beforeSend: ajaxSend,
+                data: {
+                    'langpair': curSrcLang + '|' + curDstLang,
+                    'word': fromWord,
+                    'newWord': toWord,
+                    'context': context,
+                    'g-recaptcha-response': recaptchaResponse
+                },
+                success: function (data) {
+                    $('#suggestedWordInput').tooltip('hide');
+                    $('#suggestedWordInput').tooltip('destroy');
+                    $('#suggestedWordInput').val('');
+
+                    $('#wordSuggestModal').modal('hide');
+                },
+                error: function (data) {
+                    data = $.parseJSON(data.responseText);
+                    $('#suggestedWordInput').tooltip('destroy');
+                    $('#suggestedWordInput').tooltip({
+                        'title': (data['explanation'] ? data['explanation'] : 'An error occurred'),
+                        'trigger': 'manual',
+                        'placement': 'bottom'
+                    });
+                    $('#suggestedWordInput').tooltip('show')
+                    setTimeout(function() {
+                        $('#suggestedWordInput').tooltip('hide');
+                        $('#suggestedWordInput').tooltip('destroy');
+                    }, 3000);
+                },
+                complete: function(){
```

Comment:
This should be `function ()`, not `function()`. Unfortunately this needs to be fixed in a bunch of places :( Fortunately, it's quite easy to do :)